src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+
+    duration=$SECONDS
+    echo "Uploaded ${total_files} ploidy and base count files in $(($duration / 60))m$(($duration % 60))s"
+
+    for interval in $(find /home/dnanexus/inputs/scatter-dir -type f -name "scattered.interval_list"); do
+        job_name=$(grep -Po 'chr[0-9XY]+' <<< "$interval")
+        interval_file=$(dx upload --brief "$interval")
+
+        # Bump instance type up for large interval lists, set from dxapp.json
+        interval_num=$(grep -v "^@" $interval | wc -l)
+        if [ $interval_num -gt 15000 ]; then
+            instance="$max_sub_job_instance"
```

Shellcheck

[warning] max_sub_job_instance is referenced but not assigned.

(SC2154)